engine/crates/integration-tests/setup-local-dynamo.sh

```diff
@@ -0,0 +1,7 @@
+dy --region local admin create table database --keys __pk __sk
```

Comment:
Could this happen automatically when we start the docker image?

Reply:
Can we still just run:

```console
> docker compose up
> cargo nextest run
```

and be done with it, or do we always need to run this script before the tests?

Reply:
yeah, you just need to start docker & run the tests.

you know what, i'm getting rid of this script - it's presence is probably more confusing than it's worth